crates/bevy_ui/src/geometry.rs

```diff
@@ -307,9 +305,9 @@ impl UiRect {
     /// #
     /// let ui_rect = UiRect::bottom(Val::Px(10.0));
     ///
-    /// assert_eq!(ui_rect.left, Val::Undefined);
-    /// assert_eq!(ui_rect.right, Val::Undefined);
-    /// assert_eq!(ui_rect.top, Val::Undefined);
+    /// assert_eq!(ui_rect.left, Val::Px(0.));
+    /// assert_eq!(ui_rect.right, Val::Px(0.));
+    /// assert_eq!(ui_rect.top, Val::Px(0.));
```

Comment:
GitHub won't comment me there directly, but can we maybe add a line to the documentation to make it explicit that all other values will be zero?

So something like:

```
Creates a new [`UiRect`] where `bottom` takes the given value.
All other values will be zero.
```

And then for the other methods (e.g. `top`) as well.

Reply:
Yes that makes sense and the same should be done for the `Size` helper functions too.